pkg/networkmanager/firewall.jsx

```diff
@@ -538,14 +550,22 @@ class RemoveServicesModal extends React.Component {
                 .filter(z => firewall.zones[z].services.indexOf(this.props.service) !== -1);
         this.state = {
             zones: this.zonesWithService.length === 1 ? this.zonesWithService : [],
+            dialogError: undefined,
```

Comment:
There is no need to initialize variables to undeifned. I was doing in the past, now I tend to skip them to reduce the code. Again it's code style preference. You can keep it as is, I just want to make sure it's done on the purpose.